.circleci/config.yml

```diff
@@ -169,8 +169,10 @@ jobs:
       - run:
           name: Run backend tests
           command: |
-            python -m scripts.run_backend_tests --generate_coverage_report --exclude_load_tests
-
+            files_changed=$(git diff --name-only HEAD...$TRAVIS_BRANCH)
+            if [ "$files_changed" != "AUTHORS CHANGELOG CONTRIBUTORS core/templates/pages/about-page/about-page.constants.ts" ]; then
+              python -m scripts.run_backend_tests --generate_coverage_report --exclude_load_tests
+            fi
```

Comment:
I'd recommend turning the files change to a variable so that we can add more files in the future. Also, do try to move the logic up right below `checkout` stage.

Reply:
Updated.